### ISSUE TITLE
Raspberry Pi docker not starting

### ISSUE THREAD

**Issue by WombleWoo7547** [docker/for-linux#1370]
Docker is not starting on the raspberry pi 4 4GB model running RPiOS Bullsye ARMv7l with kernel 5.10.92-v7l+.

The error journalctl -xe is giving:

```
░░
░░ Automatic restarting of the unit docker.service has been scheduled, as the result for
░░ the configured Restart= setting for the unit.
Mar 20 13:52:33 raspberrypi systemd[1]: Stopped Docker Application Container Engine.
░░ Subject: A stop job for unit docker.service has finished
░░ Defined-By: systemd
░░ Support: https://www.debian.org/support
░░
░░ A stop job for unit docker.service has finished.
░░
░░ The job identifier is 2254 and the job result is done.
Mar 20 13:52:33 raspberrypi systemd[1]: docker.service: Start request repeated too quickly.
Mar 20 13:52:33 raspberrypi systemd[1]: docker.service: Failed with result 'exit-code'.
░░ Subject: Unit failed
░░ Defined-By: systemd
░░ Support: https://www.debian.org/support
░░
░░ The unit docker.service has entered the 'failed' state with result 'exit-code'.
Mar 20 13:52:33 raspberrypi systemd[1]: Failed to start Docker Application Container Engine.
░░ Subject: A start job for unit docker.service has failed
░░ Defined-By: systemd
░░ Support: https://www.debian.org/support
░░
░░ A start job for unit docker.service has finished with a failure.
░░
░░ The job identifier is 2254 and the job result is failed.
Mar 20 13:52:33 raspberrypi systemd[1]: docker.socket: Failed with result 'service-start-limit-hit'.
░░ Subject: Unit failed
░░ Defined-By: systemd
░░ Support: https://www.debian.org/support
░░
░░ The unit docker.socket has entered the 'failed' state with result 'service-start-limit-hit'.
```

The error dockerd command gives:

```INFO[2022-03-20T13:55:48.303132707Z] Starting up
INFO[2022-03-20T13:55:48.308895004Z] parsed scheme: "unix"                         module=grpc
INFO[2022-03-20T13:55:48.308974818Z] scheme "unix" not registered, fallback to default scheme  module=grpc
INFO[2022-03-20T13:55:48.309064614Z] ccResolverWrapper: sending update to cc: {[{unix:///run/containerd/containerd.sock  <nil> 0 <nil>}] <nil> <nil>}  module=grpc
INFO[2022-03-20T13:55:48.309121613Z] ClientConn switching balancer to "pick_first"  module=grpc
INFO[2022-03-20T13:55:48.314067344Z] parsed scheme: "unix"                         module=grpc
INFO[2022-03-20T13:55:48.314142732Z] scheme "unix" not registered, fallback to default scheme  module=grpc
INFO[2022-03-20T13:55:48.314204195Z] ccResolverWrapper: sending update to cc: {[{unix:///run/containerd/containerd.sock  <nil> 0 <nil>}] <nil> <nil>}  module=grpc
INFO[2022-03-20T13:55:48.314242898Z] ClientConn switching balancer to "pick_first"  module=grpc
ERRO[2022-03-20T13:55:48.322190915Z] failed to mount overlay: no such device       storage-driver=overlay2
ERRO[2022-03-20T13:55:48.322556171Z] exec: "fuse-overlayfs": executable file not found in $PATH  storage-driver=fuse-overlayfs
ERRO[2022-03-20T13:55:48.332052674Z] AUFS was not found in /proc/filesystems       storage-driver=aufs
ERRO[2022-03-20T13:55:48.338073469Z] failed to mount overlay: no such device       storage-driver=overlay
WARN[2022-03-20T13:55:48.350835718Z] Unable to find memory controller
INFO[2022-03-20T13:55:48.351669137Z] Loading containers: start.
WARN[2022-03-20T13:55:48.361853318Z] Running modprobe bridge br_netfilter failed with message: modprobe: WARNING: Module bridge not found in directory /lib/modules/5.10.92-v7l+
modprobe: WARNING: Module br_netfilter not found in directory /lib/modules/5.10.92-v7l+
, error: exit status 1
WARN[2022-03-20T13:55:48.373994018Z] Running iptables --wait -t nat -L -n failed with message: `iptables/1.8.7 Failed to initialize nft: Protocol not supported`, error: exit status 1
WARN[2022-03-20T13:55:48.400695358Z] Failed to read iptables version: exit status 1
INFO[2022-03-20T13:55:48.607227794Z] stopping event stream following graceful shutdown  error="<nil>" module=libcontainerd namespace=moby
failed to start daemon: Error initializing network controller: error obtaining controller instance: failed to create NAT chain DOCKER: iptables failed: iptables -t nat -N DOCKER: iptables/1.8.7 Failed to initialize nft: Protocol not supported
 (exit status 1)
```

**Comment by WombleWoo7547** [comment]
It is using 74MB RAM idle

**Comment by WombleWoo7547** [comment]
Found the issue! All I needed to do was a reboot. Fixed #1370
